dvc/remote/base.py

```diff
@@ -878,8 +878,8 @@ def _checkout_file(
         """The file is changed we need to checkout a new copy"""
         cache_info = self.checksum_to_path_info(checksum)
         if self.exists(path_info):
-            logger.warning(
-                "data '{}' exists. Removing before checkout.", path_info
+            logger.info(
+                "The file '{}' already exists. It will be replaced.", path_info
             )
```

Comment:
Not sure about converting it to info. Maybe let's remove it for good? Or make it `debug`?

Reply:
This was the result of the discussion from #2979, no?
Also: https://github.com/iterative/dvc/issues/2979#issuecomment-567301148:

Reply:
@skshetry Good point, that needs to be clarified. Left a comment there.